libs/sdk-core/src/breez_services.rs

```diff
@@ -238,6 +238,13 @@ impl BreezServices {
         Ok(())
     }
 
+    /// Configure the node
+    ///
+    /// This calls [NodeAPI::configure_node] to make changes to the active node's configuration.
+    pub async fn configure_node(&self, req: ConfigureNodeRequest) -> SdkResult<()> {
```

Comment:
I'm thinking about the case where there's multiple apps integrating the sdk. And each app has a different opinion on what the close address should be. They might pingpong to set this close address to a different value each time. 

Perhaps a comment saying this method should ideally just be called once in the lifetime of the node? You can do it on creation of the node. But ideally not after each `connect`.

Reply:
Done [fb7e777](https://github.com/breez/breez-sdk/pull/765/commits/fb7e777d8aaa9b7783d038baf759200c44256a81)